### PR TITLE
Origin/feature/share popup #8

### PR DESCRIPTION
**Pull request by anikamile**
#8
https://gyazo.com/1566ddc4adb1d04a5d3cb8c13b897f08
https://gyazo.com/d643ac4fc64a4cb01b5db322e6b4f570

### CODE REVIEW

File: lib/ui/organism/share_popup.dart [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+import 'package:flutter/material.dart';
+import 'package:share/share.dart';
+
+class SharePopupButton extends StatefulWidget {
```
Comment (matejthetree) [inline lib/ui/organism/share_popup.dart]: stateless

File: lib/ui/organism/share_popup.dart [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        title: Center(
+          child: Text("Here's Your Tree"),
+        ),
+      ),
+      body: Padding(
+        padding: EdgeInsets.only(top: 25, right: 25),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.end,
+          children: <Widget>[
+            GestureDetector(
```
Comment (matejthetree) [inline lib/ui/organism/share_popup.dart]: umjesto gesture detector `IconButton`

i vratis icon button u build funkciji

File: lib/ui/organism/share_popup.dart [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+              child: Icon(Icons.share),
+              onTap: () => Share.share(shareLink),
```
Comment (matejthetree) [inline lib/ui/organism/share_popup.dart]: direktno staviti url

**Review by matejthetree — CHANGES_REQUESTED** [review]

**Review by anikamile — COMMENTED** [review]
Made all the requested changes on share popup issue #8 file

File: lib/ui/organism/share_popup.dart [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:share/share.dart';
+
+class SharePopupButton extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Card(
```
Comment (matejthetree) [inline lib/ui/organism/share_popup.dart]: remove card

File: lib/ui/organism/share_popup.dart [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+      child: IconButton(
+        padding: const EdgeInsets.all(50.0),
```
Comment (matejthetree) [inline lib/ui/organism/share_popup.dart]: remove padding

**Review by matejthetree — CHANGES_REQUESTED** [review]
add test page, and test route

File: lib/ui/organism/share_popup.dart (hunk 4 above)
Comment (anikamile) [reply lib/ui/organism/share_popup.dart]: I did.

File: lib/ui/organism/share_popup.dart (hunk 5 above)
Comment (anikamile) [reply lib/ui/organism/share_popup.dart]: I removed padding.

**Review by anikamile — COMMENTED** [review]
#8
https://gyazo.com/85ddf36a436a9452862945c30dca4c4b
https://gyazo.com/2c8ddeb4eec179bef648b043e4399be9

**Pull request closed without merging**